ts/parts/OrdinalAxis.ts

```diff
@@ -355,11 +355,12 @@ class OrdinalAxis {
         SeriesClass: typeof Series
     ): void {
 
+        AxisClass.keepProps.push('ordinal');
```

Comment:
I don't see a reason to keep `ordinal` prop.

Reply:
So we instantiate it only ones per axis instance and not with every axis init: https://github.com/highcharts/highcharts/blob/5133f3e50d90fac52261571ac43fd0c76a076de5/ts/parts/OrdinalAxis.ts#L939 Condition is missing for that though.

Reply:
I added the condition for ordinal composition.